.github/workflows/check-lambda-js.yml

```diff
@@ -36,28 +37,30 @@ jobs:
       - name: Setup Node
```

Comment:
# Accessibility: Keyboard Navigation

On Safari, the menu is not navigable by keyboard at all.

Reply:
I think this might also be the longstanding Safari issue of not allowing full tab nav by default (same as above). Though I think this one only relies on the MacOS System Setting.